app/presenters/specialist_document_presenter.rb

```diff
@@ -85,6 +85,10 @@ def images
     }
   end
 
+  def show_metadata_block?
+    content_item.dig("links", "finder", 0, "details", "show_metadata_block")
```

Comment:
We should probably have added this value to the `Finder` model, and then could have referred to it wirh `finder.show_metablock`